pkg/crc/preflight/preflight_linux_test.go

```diff
@@ -0,0 +1,23 @@
+package preflight
+
+import (
+	"testing"
+
+	"github.com/code-ready/crc/pkg/crc/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCountConfigurationOptions(t *testing.T) {
+	cfg := config.New(config.NewEmptyInMemoryStorage())
+	RegisterSettings(cfg)
+	options := len(cfg.AllConfigs())
+	assert.True(t, options == 40 || options == 32)
+}
+
+func TestCountPreflights(t *testing.T) {
+	assert.Len(t, getPreflightChecksForDistro("rhel", false), 21)
```

Comment:
better to use a type or something that acts like an enum. using strings is not that 'safe'

Reply:
done